.github/workflows/celo-monorepo.yml

```diff
@@ -8,10 +8,12 @@ on:
   push:
     branches:
       - master
+      - changeset-release/master
   pull_request:
     branches:
       - master
-      - prerelease/*
+      - 'releases/**'
+      - 'prereleases/**'
```

Comment:
Did you intentionally go from `release` to `releases` (plural)? This does not match the past naming.

Reply:
nope good catch